app/src/main/java/com/safetyheads/akademiaandroida/AndroidAcademyApplication.kt

```diff
@@ -33,4 +36,13 @@ class AndroidAcademyApplication: Application() {
         viewModel{ SplashScreenViewModel(get()) }
         viewModel{ DropDownListViewModel(get()) }
     }
+
+    private val retrofitModule = module {
+        // Singleton Retrofit Interceptor and ApiClient
+        single { ApiClient() }
+        single { RetrofitInterceptor() }
```

Comment:
This definitions is useless, it wouldn't be provided via DI container. Creation in `ApiClient` is enought.